source/includes/connect/atlas.cpp

```diff
@@ -0,0 +1,34 @@
+#include <mongocxx/client.hpp>
+#include <mongocxx/uri.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/exception/exception.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
+
+int main()
+{
+    // Replace the placeholder with your Atlas connection string
```

Comment:
```suggestion
    mongocxx::instance();
    // Replace the placeholder with your Atlas connection string
```

Suggest including `mongocxx::instance()` in examples that are shown to the user as a complete file.

I expect users may copy-paste this example. Omitting the `mongocxx::instance()` may result in a difficult-to-diagnose error. Updating `<connection string>` and running locally results in:

```
Segmentation fault: 11  ./cmake-build/main.out
```

If suggestion is accepted, also add `#include <mongocxx/instance.hpp>` at the top of the file.

Suggestion applies to other code examples.

Reply:
Since the docs have so many code examples, we had hoped to avoid including mongocxx::instance in every example. Instead, we included an admonition about it on line 26 of connection-targets.txt.

Maybe it makes sense to include an instance in all code examples that are "complete"--i.e., those that include a main() method and that we expect users to copy _in toto_ to their own applications. What do you think?

Reply:
> Maybe it makes sense to include a mongocxx::instance in all code examples that are "complete"--i.e., those that include a main() method and that we expect users to copy in toto to their own applications

That agrees with my viewpoint. Suggested addition to [C/C++ Driver Code Example Patterns and Style Reference](https://docs.google.com/document/d/1wBsUnT69a7XNTUOZd1agQae_pL1fREl-mcg34QgYaCs/edit#heading=h.2fm832wk86xh).